docs/testingplatform/Source/TestingPlatformExplorer/TestingFramework/TestingFramework.Attributes.cs

```diff
@@ -6,6 +6,12 @@ namespace TestingPlatformExplorer.TestingFramework;
 [AttributeUsage(AttributeTargets.Method)]
 public class SkipAttribute : Attribute
 {
+    public string Reason { get; private set; }
```

Comment:
@MarcoRossignoli this should be a readonly property, could you please do a follow-up.

Reply:
FYI it's the sample code it's not shipped.

Reply:
Ah sorry didn't pay attention.